templates/boilerplate-template/scripts/move/get_abi.js

```diff
@@ -0,0 +1,27 @@
+require("dotenv").config();
+const axios = require("axios");
+const fs = require("node:fs");
+
+const modules = [
+  { address: process.env.VITE_MODULE_ADDRESS, name: "message_board" },
+  { address: "0x1", name: "coin" },
+];
+
+async function getAbi() {
+  modules.forEach((module) => {
+    const url = `https://fullnode.${process.env.VITE_APP_NETWORK}.aptoslabs.com/v1/accounts/${module.address}/module/${module.name}`;
+    axios
```

Comment:
wait how come it works? `axios` is not part in package.json. Also, any reason not to use the native `fetch`?

Reply:
it works, probably some dep includes axios

Reply:
removed, use fetch instead